add SG and PS post (#572)
usegalaxy-eu/website: _posts/2021-01-18-sg-ps-update.md

---
site: [freiburg]
title: Major update to SearchGUI and PeptideShaker
tags:
- tools
---

After 3 years of work, [SearchGUI (SG)](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/search_gui/),
a tool that performs protein identification using various search engines, and [PeptideShaker (PS)](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/peptide_shaker/)

>  for protein identification which uses SearchGUI results have been deeply updated and released in a new major versions 4.0.7 and 2.0.5 respectively.

Reviewer: ```suggestion
 for protein identification (which uses SearchGUI results) have been deeply updated and released in new major versions: 4.0.7 and 2.0.5, respectively.
```

Furthermore, two new utility tools have been added to make the overall workflow more flexible:

> * [Identification Parameters](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/ident_params/) for creating parameter files reusable by both SG and PS

Reviewer: ```suggestion
* [Identification parameters](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/ident_params/) for creating parameter files reusable by both SG and PS.
```

> * [FastaCLI](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/fasta_cli/) which appends decoy sequences to a fasta file in a format readable by SG and PS

Reviewer: ```suggestion
* [FastaCLI](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/fasta_cli/), which appends decoy sequences to a FASTA file in a format readable by SG and PS.
```

> The number of changes and improvements made is very large, a few highlights:

Reviewer: ```suggestion
The number of changes and improvements made is large, a few highlights:
```

> ### SearchGUI:

Reviewer: ```suggestion
### SearchGUI
```

> * Support for spectrum files in mzML format

Reviewer: ```suggestion
* Support for spectrum files in mzML format.
```

> * Search engines updated to last versions. Options updated accordingly

Reviewer: ```suggestion
* Search engines updated to the most recent versions. Options updated accordingly.
```

> * Search engines updated to last versions. Options updated accordingly

Reviewer: I am not sure that 'Options updated accordingly' means

Author: Parameters of the search engine

> * FASTA file is now out of the search parameters

Reviewer: ```suggestion
* FASTA file is now out of the search parameters.
```

> * Many more supported search engines parameters

Reviewer: ```suggestion
* Many more supported search engines parameters.
```

> * more [detailed release notes](http://compomics.github.io/projects/searchgui/wiki/ReleaseNotes)

Reviewer: ```suggestion
* More [detailed release notes](http://compomics.github.io/projects/searchgui/wiki/ReleaseNotes).
```

### PeptideShaker

> * Support for spectrum files in mzML format

Reviewer: ```suggestion
* Support for spectrum files in mzML format.
```

> * Internal refactoring, new internal db and more efficient data management

Reviewer: ```suggestion
* Internal refactoring, new internal database and more efficient data management.
```

> * Reduced file sizes and memory load

Reviewer: ```suggestion
* Reduced file sizes and memory load.
```

> * more [detailed release notes](http://compomics.github.io/projects/peptide-shaker/wiki/ReleaseNotes )

Reviewer: ```suggestion
* More [detailed release notes](http://compomics.github.io/projects/peptide-shaker/wiki/ReleaseNotes).
```

> Thanks Carlos Horro Marcos for your awesome work on the Galxy integration.

Reviewer: ```suggestion
Thanks, Carlos Horro Marcos for your awesome work on the Galaxy integration.
```
